addons/website/views/website_templates.xml

```diff
@@ -731,12 +731,15 @@
                     <h2 class='mt32'>Installed Localizations / Account Charts</h2>
                     <dl class="dl-horizontal" t-foreach="l10n" t-as="app">
                         <dt>
-                            <a t-attf-href="https://www.odoo.com/page/accounting/#{app.name}">
-                                <t t-raw="app.shortdesc"/>
+                            <a t-att-href="app.website" t-if="app.website">
+                                <t t-esc="app.shortdesc"/>
+                            </a>
+                            <a t-attf-href="https://www.odoo.com/page/accounting/#{app.name}" t-if="not app.website">
```

Comment:
`t-else`?
In master this could just be a single `a`, I guess you wrote it like that to not break any potential xpath?
```suggestion
                            <a t-att-href="app.website or 'https://www.odoo.com/page/accounting/' + app.name">
```

Reply:
Jep, I did, but I kinda doubt that someone is inheriting this as mostly you will deactivate it or do not know that it exists 😉